src/resources/4_0_0/profiles/observation/resolver.js

```diff
@@ -3,87 +3,114 @@
  * @static
  * @summary Observation resolver.
  */
-module.exports.getObservation = function getObservation(
+module.exports.getObservation = async function getObservation(
 	root,
 	args,
 	context = {},
 	info,
 ) {
-	let { server, version, req, res } = context;
-	return {};
+		try {
+			const { server: { model, app }, version, req, res } = context;
+			return (await model.observations.getOne(args));
+		} catch (e) {
+			console.log('Observation error: ', e);
+		}
```

Comment:
I have suggestion for your code. Please make it small and clean. If you don't use any variable, you should delete it. @Kolbein @JonasBacker 


```
module.exports.getObservation = async function (_, args, context = {}) {
	const { server: { model } } = context;
	return await model.observations.getOne(args);
}
```
or 

```
module.exports.getObservation = async function (_, args, context = {}) {
	return await context.server.model.observations.getOne(args);
}
```